candle-pyo3/py_src/candle/__init__.pyi

```diff
@@ -67,7 +67,7 @@ class u8(DType):
     pass
 
 @staticmethod
-def zeros(shape: Sequence[int], dtype: Optional[DType] = None, device: Optional[Device] = None) -> Tensor:
+def zeros(shape: Shape, dtype: Optional[DType] = None, device: Optional[Device] = None) -> Tensor:
```

Comment:
Why isn't this one a `*shape` too?

Reply:
Yup, i provided the wrong typehint there.